api_tests/tests/orderbook.ts

```diff
@@ -10,21 +10,40 @@ import { getIdentities } from "../src/actors/defaults";
 
 // todo: move test initialisation into single mega function to reduce noise
 describe("orderbook", () => {
+    // direct quote
+    // BTC/DAI
+    // position: buy
+    // quote: 1.71 (DAI/BTC)
```

Comment:
I finally worked out what you mean here 'DAI per BTC', this is a really confusing usage because two assets with a slash is used two lines above to mean a totally different thing i.e., the currency pair (is is customary).

Please write something like: 'quoted in units of DAI'

Reply:
or  'DAI per unit of BTC'